client/modules/core/components/app-title/app-title-component.js

```diff
@@ -23,15 +25,21 @@ const useStyles = makeStyles(({palette, spacing}) => ({
   },
 }));
 
-function AppTitle({appTitle, goToDashboard, signIn, signOut, toggleTheme, user}) {
+function AppTitle({appTitle, goToDashboard, signIn, signOut, toggleTheme, user, history}) {
   const classes = useStyles();
+
+  function signUserOut() {
```

Comment:
```suggestion
  function handleSignOut() {
```
I think calling it handleSignOut is more in keeping with the convention for doing a set of actions on a button click.